services/build-status.js

```diff
@@ -21,8 +21,9 @@ const redStatuses = [
   'infrastructure_failure',
 ]
 
+const yellowStatuses = ['building']
+
 const otherStatuses = [
-  'building',
```

Comment:
There's a lot of badges using this helper and I think this change will have negative impacts on other badges. If you'd prefer to have a `yellow` color for the `building` status in Netflify then IMO you should override that color directly within the Netflify service vs. changing it here

Reply:
Also, this is causing the build to fail.

```suggestion
  'building',
```